elytron/src/test/resources/org/wildfly/extension/elytron/elytron-transformers-1.2.xml

```diff
@@ -16,6 +16,9 @@
     <security-realms>
         <identity-realm name="identity" identity="identity" />
     </security-realms>
+    <credential-security-factories>
```

Comment:
same remark as above on the xml namespace

Reply:
No as I already said above - the version in this name is the version we are transforming to.